src/classBased/Components/TodoItem.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable */
```

Comment:
Please try not to disable linters, it would be better if you fix linter errors instead